### PR TITLE
Add RHEL 8.10 to the dropdown

### PR DESCRIPTION
**Pull request by amirfefer**
# Description

Add RHEL 8.10 support.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -86,9 +86,11 @@ export const releaseMapper = {
   'rhel-86': 'Red Hat Enterprise Linux (RHEL) 8.6',
   'rhel-85': 'Red Hat Enterprise Linux (RHEL) 8.5',
   'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
+  'rhel-81': 'Red Hat Enterprise Linux (RHEL) 8.1',
```
Comment (adarshdubey-star) [inline src/constants/index.js:89]: Shouldn't it be ?
```suggestion
  'rhel-81': 'Red Hat Enterprise Linux (RHEL) 8.10',
```

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)

File: src/constants/index.js (hunk 1 above)
Comment (loadtheaccumulator) [reply src/constants/index.js:89]: We can discuss deprecating minor versions, but we should keep 8.1 and add 8.10 here and in the supportedReleases below.

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]
suggesting adding 8.10 instead of replacing.

File: src/constants/index.js [hunk 2]
```diff
@@ -86,9 +86,11 @@ export const releaseMapper = {
...
 };
 
 export const supportedReleases = [
+  'rhel-81',
```
Comment (loadtheaccumulator) [inline src/constants/index.js:93]: This should reflect 8.10 instead of 8.1

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]

File: src/constants/index.js (hunk 1 above)
Comment (lzap) [reply src/constants/index.js:89]: The key must be: `rhel-8.10` (yeah it is different).

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)

**Comment by amirfefer** [comment]
Screenshot 2024-09-11 at 15 41 53

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of react-router-dom (5.3.4) is older than required version (6.0.0)
* Installed version of @redhat-cloud-services/frontend-components-config (4.7.3) is older than required version (6.0.0)

**Review by lzap — APPROVED** [review]

**Pull request merged**